src/main/java/ca/corefacility/bioinformatics/irida/service/remote/ProjectSynchronizationService.java

```diff
@@ -374,6 +377,24 @@ private void syncProject(Project project) {
 			}
 		}
 
+		//list the fast5 files from the remote api
+		List<Fast5Object> fast5FilesForSample = fast5ObjectRemoteService.getFast5FilesForSample(sample);
```

Comment:
Note the try/catch around this block in the assemblies section below.  It should be added in case the host system doesn't support fast5 data.

Reply:
Added in ccf8d68de322f55c99d3283986c593dd49df391e